tests/repack-kernel/task.yaml

```diff
@@ -0,0 +1,35 @@
+summary: tests for the repack-kernel tool
+
+# ubuntu-core-initramfs seems to be only available on ubuntu-20.04
+systems: [ubuntu-20.04-64]
+
+execute: |
+    # install dependencies
+    apt update -yqq
+    apt install snapd
+
+    # download the kernel snap
+    snap download pc-kernel --channel=20/stable --basename=upstream-pc-kernel
+
+    # setup dependencies
+    repack-kernel setup
+
+    # extract the kernel snap, including extracting the initrd from the kernel.efi
+    kerneldir=/tmp/kernel-workdir
+    repack-kernel extract "upstream-pc-kernel.snap" "$kerneldir"
+
+    # cull modules and firmware
+    repack-kernel cull-modules "$kerneldir"
+    repack-kernel cull-firmware "$kerneldir"
+
+    # repack the initrd into the kernel.efi
+    repack-kernel prepare "$kerneldir"
```

Comment:
could you please include some error scenarios, when you call:
1. the command does not exist
2. the kerneldir parameter is not passed
3. the help is returned with no command, with -h and --help